ibc-primitives/src/lib.rs

```diff
@@ -25,8 +25,9 @@ pub use traits::*;
 mod types;
 pub use types::*;
 
-/// Re-export of common proto types from the `ibc-proto` crate.
+/// Re-exports google proto types and `Protobuf` trait from the `ibc-proto-rs`
+/// crate.
 pub mod proto {
-    pub use ibc_proto::google::protobuf::Any;
+    pub use ibc_proto::google::protobuf::*;
```

Comment:
the scope of `ibc_proto::google::protobuf::*` is too broad. The structs other than `Any`, `Timestamp`, and `Duration` are for parsing protobuf files. Do we really need them? 

```suggestion
    pub use ibc_proto::google::protobuf::{Any, Duration, Timestamp};
```

Reply:
I was under the impression that a few types are exposed by `ibc_proto::google::protobuf` not all. Thank you for pointing that out 👌🏻